### DISCUSSION TITLE
#_TASKSIGNUPCANCEL_LINK how to use it ?

### DISCUSSION THREAD

**Discussion by Zip56400** [liedekef/events-made-easy#430] [Q&A] [answered]
Good evening,

I try to give volunteers the opportunity to opt out of a task, but when integrating the placeholder #_TASKSIGNUPCANCEL_LINK into the task registration confirmation template, the email sent contains a link to a list general of events, and does not give the possibility of unsubscribing from the task. I think I'm not doing things correctly, could you help me ?
Tanks in advance,
Zip

**Comment by liedekef** [comment, marked as answer]
There was indeed a bug. This will be fixed in the next version, but for completeness:
https://github.com/liedekef/events-made-easy/commit/bc2059e7e5616f377f85be711cc85e8b8cf62770

**Reply by Zip56400** [reply]
Thank's a lot !
